app/Dockerfile

```diff
@@ -19,8 +19,10 @@ COPY ./setup.py /app/setup.py
 
 # Style file for AGU journal pdf
 RUN mkdir -p /usr/share/texlive/texmf-dist/tex/latex/agujournal2019/
-RUN chmod 644 ./app/pdf/agujournal2019.cls
+#RUN chmod 644 ./app/pdf/agujournal2019.cls
```

Comment:
Is this not needed anymore, should we remove?

Reply:
Indeed no longer needed. I'm terrible at removing commented-out code before comitting